package.json

```diff
@@ -3,21 +3,27 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "4.2.2",
     "@antv/g2plot": "1.1.21",
+    "@reduxjs/toolkit": "1.4.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "antd": "4.5.2",
     "axios": "0.19.2",
```

Comment:
it occurred we are using fetch, can you migrate to simple fetch for us to pick up you plugin easier?

Reply:
migrated